tools/accuracy_checker/accuracy_checker/launcher/model_conversion.py

```diff
@@ -177,24 +184,35 @@ def exec_mo_binary(args, timeout=None, should_log_cmd=False):
     return subprocess.run(args, check=False, timeout=timeout)
 
 
-def set_path_to_custom_operation_configs(mo_params, framework, tf_custom_op_config_dir, mo_path):
+def set_path_to_custom_operation_configs(
+        mo_params, framework, tf_custom_op_config_dir, transformations_config_dir, mo_path
+):
     if framework.name != 'tf':
         return mo_params
 
-    config_path = mo_params.get('tensorflow_use_custom_operations_config')
-    if not config_path:
+    tf_custom_op_config_path = mo_params.get('tensorflow_use_custom_operations_config')
+    transformations_config = mo_params.get('transformations_config')
+    if not tf_custom_op_config_path and not transformations_config:
         return mo_params
 
     if tf_custom_op_config_dir:
-        tf_custom_op_config_dir = Path(tf_custom_op_config_dir)
+        prefix_dir = Path(tf_custom_op_config_dir)
+    elif transformations_config_dir:
+        prefix_dir = Path(transformations_config_dir)
     else:
-        tf_custom_op_config_dir = Path('/').joinpath(*mo_path.parts[:-1]) / 'extensions' / 'front' / 'tf'
+        prefix_dir = Path('/').joinpath(*mo_path.parts[:-1]) / 'extensions' / 'front' / 'tf'
 
-    config_path = Path(config_path)
-    if not config_path.is_absolute():
-        config_path = tf_custom_op_config_dir / config_path
+    if tf_custom_op_config_path:
+        config_path = Path(tf_custom_op_config_path)
+        if not config_path.is_absolute():
+            config_path = prefix_dir / config_path
+            mo_params['tensorflow_use_custom_operations_config'] = str(get_path(config_path))
+        return mo_params
```

Comment:
This `return` is IMO not a good thing. It means that if both the TF config and generic config are specified, the latter will be silently ignored, which is surprising behavior. AC should either pass both options to MO or complain and refuse to do anything.

Reply:
options separated, I do not want be responsible on verifying MO arguments (it is their obligation)